### PR TITLE
Update Dutch translation

### PR DESCRIPTION
**Pull request by flipflop97**

### CODE REVIEW

File: po/nl.po [hunk 1]
```diff
@@ -586,200 +700,240 @@ msgstr "LMS-formaat"
...
+"@bertob</li><li>Export palettes as Adobe Swatch Exchange (ASE), hex, PAL, "
+"Paint.NET (txt) files</li><li>Minor changes to the store metadata</li></"
+"ul><p>Changed:</p>\n"
+"<ul><li>Hunter-Lab now displays 0 instead of NaN</li><li>Improved tooltips "
+"for copying colors</li><li>Palette window now uses a button with a more "
+"explicit text instead of an icon</li><li>Reworded project description</"
+"li><li>Updated all screenshots</li></ul><p>Fixed:</p>\n"
+"<ul><li>Separator is now hidden when the history list is hidden</"
+"li><li>Improved editing experience when change the Hex color/name</li></ul>"
 msgstr ""
```
Comment (FineFindus) [inline po/nl.po:870]: I think you missed one here

**Review by FineFindus — CHANGES_REQUESTED** [review]
Found a missing translation, looks good otherwise.

File: po/nl.po (hunk 1 above)
Comment (flipflop97) [reply po/nl.po:870]: Ah, I'm sorry. I might have done it a little too quickly. I'll try to finish that last one soon!

Comment (flipflop97) [reply po/nl.po:870]: Done!

**Comment by FineFindus** [comment]
Thanks!

**Pull request merged**